docs/tutorials/excel-tutorial-create-custom-functions.md

```diff
@@ -221,4 +221,4 @@ You can prevent this conflict by [clearing the Office cache](../testing/clear-ca
 
 ## Next steps
```

Comment:
Maybe another bug/PR to track, but adding the clock function in the previous steps causes an error. The yo office project already has this code. so we need a different streaming example (or perhaps take this one out?)

Reply:
Thanks! Created a [work item](https://office.visualstudio.com/OC/_workitems/edit/7718962) to track this task separately.